scripts/strings/portuguese-ptpt/sys_notify.lua

```diff
@@ -0,0 +1,67 @@
+-- OPENTOMB SYSTEM CONSOLE NOTIFICATIONS
```

Comment:
console output and system messages now is hardcoded and English only to avoid broken / empty debug output in case of script crash or other heavy bugs...